src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -524,7 +474,7 @@ const useGroupInfo = () => {
     setSelectedUser,
     openChatMessage,
     generateSystemChat,
-    setNewParticipan,
+    setNewParticipant,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setNewParticipant` is now being used instead of `setNewParticipan`. This is a good change as it corrects the typo in the previous function name.

```diff
- setNewParticipan,
+ setNewParticipant,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->